task4/func.js

```diff
@@ -0,0 +1,21 @@
+function Person(name,age) {
+    this.name = name;
+    this.age = age;
+}
+
+Person.prototype.introduce = function () {
+    return 'My name is ' + this.name + ' and I am ' + this.age;
+}
+
+function myNew(type, ...args) {
+    // create new empty object
+    let instance = {};
+    // assign prototype to new instance -> Person
+    instance.__proto__ = type.prototype;
```

Comment:
It's not a good idea to use `__proto__`
It's like private property and it's better to avoid it at all.
Object.create is much better :)